packages/mui-material/src/InputAdornment/InputAdornment.js

```diff
@@ -45,7 +45,6 @@ const InputAdornmentRoot = styled('div', {
   overridesResolver,
 })(({ theme }) => ({
   display: 'flex',
-  height: '0.01em', // Fix IE11 flexbox alignment. To remove at some point.
```

Comment:
Introduced in https://github.com/mui/material-ui/pull/12975

Tested it in Chrome, Firefox, and Safari. Screenshot in Chrome:

<img width="400" alt="Screenshot 2024-05-23 at 16 16 02" src="https://github.com/mui/material-ui/assets/16889233/5ac4d45e-c4a1-42ae-afa7-424091ea122e">